src/test/java/com/purbon/kafka/topology/AccessControlManagerTest.java

```diff
@@ -170,23 +149,16 @@ public void testConsumerAclsAtTopicLevel() {
 
   @Test
   public void newProducerACLsCreation() {
-
-    List<Producer> producers = new ArrayList<>();
-    producers.add(new Producer("User:app1"));
-    Project project = new ProjectImpl();
-    project.setProducers(producers);
-
     Topic topicA = new TopicImpl("topicA");
-    project.addTopic(topicA);
-
-    Topology topology = new TopologyImpl();
-    topology.addProject(project);
+    TestTopologyBuilder builder =
+        TestTopologyBuilder.createProject().addTopic(topicA).addProducer("User:app1");
+    List<Producer> producers = new ArrayList<>(builder.getProducers());
```

Comment:
help me understand, why here is better new ArrayList and in other places asLists or singletonList?

Reply:
It is aweful indeed ;-) 

If we were on Java 10 then I would have used List.copyOf()... But I see that we already have Guava in the classpath so I can change to Lists.newArrayList() if you like that better. 

However it would be great to "move" KTB to Java 10 source. A lot of small code improvements here and there. :-)

Reply:
I would love as well, but sadly this is still used in very old setups ... for the moment hard to move out of java8.... sorry man.

Reply:
Can change to newArrayList() as it is 5% nicer :-) ?